### DISCUSSION TITLE
Need help with text

### DISCUSSION THREAD

**Discussion by donnaran** [ExperienceLovelace/ha-floorplan#372] [General]
need help with text_set. the bellow was working in .37 but after updating to .38 or .39 below not working any more. its doesn't update the state in SVG.   
  

> - name: home_status_messages

    entities:
       - entity: '*'
    state_action:
          action: call-service
          service: floorplan.text_set
          service_data: 
             elements: 
                - h_status_messages_state
                - fp_status_messages
             shift_y_axis: 1.5em
             text: |
                     >  
                      return  entities["sensor.fp_messages_active"].state"

**Comment by OzGav** [comment]
You will need to properly format that so we can read it

**Reply by donnaran** [reply]
i have updated in proper format above. Please let me know what is wrong with this?

**Reply by OzGav** [reply]
Yeah I don’t think we support the * entity anymore as that was causing other problems. I have never seen that `shift_y_axis` entry before so not sure if that is doing anything?

can you confirm what you are trying to do here. Are you putting the state of sensor.fp_messages_active in two places (those two elements) on the floorplan?

**Reply by donnaran** [reply]
i have sensor called sensor.fp_messages_active which i am trying to display state value  in two different FP elements h_status_messages_state and fp_status_messages.

by the look entity support for * removed in .38 & .39. If i replace the * with entity name it does update the state.

**Reply by OzGav** [reply]
Ok then is this solved by just changing the entity name?

**Reply by donnaran** [reply]
Yes sort off. I was using the entity as  * so it refresh the floorplan  i dont need to refresh the page manually.

**Reply by OzGav** [reply]
You shouldn’t need to refresh the page. If you use the entity that is the one with the text you are wanting to add to the flooprlan then as soon as that entity state changes then the floorplan will update

**Comment by donnaran** [comment]
in .37  working text element updated. like below

![image](https://github.com/ExperienceLovelace/ha-floorplan/assets/56533418/8f26714e-0897-42e1-8959-a65884beba3b)

in .38 or .39 it doesn't update the element text.

![image](https://github.com/ExperienceLovelace/ha-floorplan/assets/56533418/7c2d452f-3884-4396-976f-87860d5d7b6a)

**Reply by exetico** [reply]
In what situation did you use the `*`, and in what case do you think it's still a must have for you?
I can't see why it makes any sense to use the `**, **but maybe** I've personally missed the use-case for it.
